docs/Pflichtenheft.tex

```diff
@@ -529,7 +529,54 @@ \subsubsection{Anwendungsfalldiagramm}
 \end{center}
 
 \subsection{Dynamische Modelle}
-\Blindtext[1]
+\subsubsection{Starten der \gls{Uberwachungskonsole}}
```

Comment:
glossary in sections sollte man nicht machen. Bekomme da Warnungen im Code angezeigt, weil das PDF dann möglicherweise Probleme bekommt (Link in Kapitelname)